banks.ini

```diff
@@ -370,7 +370,7 @@ prefix   = "f47G"
 
 [bank-44]
 name     = "AIL (Terra Nova Strike Force Centuri) :p4op:"
-пфьуы    = "Terra Nova Strike Force Centuri"
+games    = "Terra Nova Strike Force Centuri"
```

Comment:
Whoops, I actually hurried while added this entry and just typed with wrong layout switched, thanks!